idd/Energy+.idd.in

```diff
@@ -76268,17 +76268,21 @@ ThermalStorage:Ice:Detailed,
   A4 , \field Outlet Node Name
        \required-field
        \type node
-  A5 , \field Discharging Curve Object Type
+  A5 , \field Discharging Curve Variable Specifications
```

Comment:
This field and "Charging Curve Variable Specifications" should have a default, or be required.
Should the curve names be required?
Any other fields that should be required or have a default?

Reply:
@mjwitte Made all four required fields.  No defaults--the user has to make some sort of decision here.